lib/package-managers/base-package-manager.js

```diff
@@ -11,11 +11,19 @@ exports.BasePackageManager = class {
   }
 
   run(command, args = [], workingDirectory = process.cwd()) {
+    const isWindows = process.platform === "win32";
+    let executable = this.getExecutablePath(workingDirectory);
+    let options = { stdio: "inherit", cwd: workingDirectory };
+    if (isWindows) {
+      executable = `"${executable}"`
+      options = { ...options, shell: true }
```

Comment:
`option.shell = true;` is much cheaper. Or even move it outside of the if.

`const options = { stdio: ..., cwd: ..., shell: isWindows };`.

Reply:
Good idea. Since there is no longer a need for a variable this way, I did as suggested within the function call parameters.